app/controllers/idv/phone_controller.rb

```diff
@@ -9,14 +9,27 @@ class PhoneController < ApplicationController
     before_action :set_idv_form
 
     def new
-      analytics.track_event(Analytics::IDV_PHONE_RECORD_VISIT)
+      async_state = step.async_state
+
+      case async_state.status
+      when :none
+        analytics.track_event(Analytics::IDV_PHONE_RECORD_VISIT)
+        render :new
+      when :in_progress
+        render :wait
```

Comment:
do we want to provide coverage for in progress and timed out?

Reply:
spec added!